src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTest/UnitTestsControl.xaml

```diff
@@ -34,14 +34,14 @@
 				<Button
 					x:Name="runButton"
 					Grid.Column="0"
-					Content="▶ Run"
```

Comment:
Hmm, why is this changed?

Reply:
part of debug code that was pushed by accident; didnt have chance to fix my pr yet (hence why it was in draft))